iset.mm

```diff
@@ -95653,14 +95654,12 @@ the previous value and the value of the input sequence (second operand).
     iseqf.2 $e |- ( ph -> M e. ZZ ) $.
     iseqf.3 $e |- ( ( ph /\ x e. Z ) -> ( F ` x ) e. S ) $.
     iseqf.4 $e |- ( ( ph /\ ( x e. S /\ y e. S ) ) -> ( x .+ y ) e. S ) $.
-    $( Range of the recursive sequence builder.  (Contributed by Jim Kingdon,
-       23-Jul-2021.) $)
+    $( Range of the recursive sequence builder.  Because this theorem
+       needlessly requires that ` S ` be a set, new proofs should use ~ iseqfcl
+       instead.  (Contributed by Jim Kingdon, 23-Jul-2021.)
+       (New usage is discouraged.) $)
     iseqf $p |- ( ph -> seq M ( .+ , F , S ) : Z --> S ) $=
-      ( vk cfv wf cv wcel wa cuz cseq wral eleq2i sylan2br iseqfn simpr adantlr
-      wfn adantr co iseqclOLD ralrimiva ffnfv sylanbrc feq2i sylibr ) AGUAPZEDE
-      FGUBZQZIEUSQAUSURUIORZUSPESZOURUCUTABCDEFGHLKBRZURSZAVCISVCFPESZIURVCJUDM
-      UEZNUFAVBOURAVAURSZTBCDEFGVAHAVGUGAEHSVGKUJAVDVEVGVFUHAVCESCRZESTVCVHDUKE
-      SVGNUHULUMOUREUSUNUOIUREUSJUPUQ $.
+      ( iseqfcl ) ABCDEFGIJLMNO $.
   $}
```

Comment:
Why not completely remove this theorem in favor of ~iseqfcl, since it is only an alias now?

Reply:
Oh, sure, I thought this already covered all usages.